src/contracts/ChainContract.ts

```diff
@@ -1,13 +1,15 @@
 import {inject, injectable} from 'inversify';
-import {Contract} from 'ethers';
+import {Contract, BigNumber} from 'ethers';
 import {TransactionResponse} from '@ethersproject/providers';
-import {ABI, ContractRegistry} from '@umb-network/toolbox';
+import {ContractRegistry} from '@umb-network/toolbox';
 import {PayableOverrides} from '@ethersproject/contracts';
 
+import newAbi from './Chain.abi.json';
```

Comment:
`newAbi` is a strange name. can we use something a bit more descriptive. just because it's new now doesn't mean it will always be new.

Reply:
I'm not sure what would be best name here, I did: 
`import chainV470abi from './ChainV4.7.0.abi.json';`